### DISCUSSION TITLE
The skywalking configuration alarm cannot be sent

### DISCUSSION THREAD

**Discussion by 1450793561** [apache/skywalking#11655] [General]
skywalking version：9.7.0
The skywalking configuration alarm cannot be sent？？？

alarm-settings.yml file As follows：
```
rules:
  endpoint_percent_rule:
    expression: avg(service_resp_time) <= 2000
    period: 10
    silence-period: 10
    message: 服务【{name}】的平均响应时间在最近10分钟内有2分钟超过1秒
    tags:
      level: WARNING
hooks:
  slack:
    default:
      is-default: true
      text-template: |-
        {
          "type": "section",
          "text": {
            "type": "mrkdwn",
            "text": ":alarm_clock: *Apache Skywalking Alarm* \n **%s**."
          }
        }
      webhooks:
      - https://qyapi.weixin.qq.com/cgi-bin/webhook/send?key=xxxxxxxx
```

When the oap service is started, you can also view the log of loading rules

<img width="1430" alt="企业微信截图_c4b2d0d8-af80-4847-b547-11da0dd5e400" src="https://github.com/apache/skywalking/assets/31468018/aabcb854-e371-4c51-b919-7bee24f1eec5">

On the ui page, you can also view the trigger rule in the alarm area

<img width="1023" alt="企业微信截图_2dde34d7-0747-4992-97eb-3cad9a844d68" src="https://github.com/apache/skywalking/assets/31468018/ec43bb31-bb8d-4043-914d-1853a84cf555">

**Comment by wu-sheng** [comment]
You are using a Slack API to forward messages to weixin.com? How is this possible to work?

**Reply by 1450793561** [reply]
Enterprise wechat robot can also use slack this?

**Reply by wu-sheng** [reply]
If you want to enterprise wechat, choose that hook. Why do you think Slack and Wechat are the same thing?

**Reply by 1450793561** [reply]
The following is OK？
```
hooks:
  WeChat:
    default:
      is-default: true
      text-template: |-
        {
          "type": "section",
          "text": {
            "type": "mrkdwn",
            "text": ":alarm_clock: *Apache Skywalking Alarm* \n **%s**."
          }
        }
      webhooks:
      - https://qyapi.weixin.qq.com/cgi-bin/webhook/send?key=xxxxx
```

**Reply by wu-sheng** [reply]
I don't know the details. Follow the docs, and that is your env. 
I am not a WeChat enterprise user, don't count on me about any experience.

**Reply by 1450793561** [reply]
Can I see logs on the oap when sending alarm notifications?

**Comment by 1450793561** [comment]
done
